pkg/kubeinteraction/labels.go

```
@@ -85,4 +86,10 @@ func AddLabelsAndAnnotations(event *info.Event, pipelineRun *tektonv1.PipelineRu
 	for k, v := range annotations {
 		pipelineRun.Annotations[k] = v
 	}
+
+	// Add annotations to PipelineRuns to integrate with Tekton Results
+	err := AddResultsAnnotation(event, pipelineRun)
+	if err != nil {
+		logger.Errorf("Failed to add results annotations to PipelineRuns with error: %w", err)
```

I think we need to handle the error here

Earlier there was no error to return from `AddLabelsAndAnnotations` so there was no return but now `AddResultsAnnotation` retruns error and if you don't handle we miss the failure

---

Yes, as commented [here](https://github.com/openshift-pipelines/pipelines-as-code/pull/1245#discussion_r1176021307), I think we may just log without returning error?
If you think otherwise, let me know.

---

we want to return the error and return it back, to the initial callers, since we have some logic to take that error and echo it in the user repo namespace

---

Hey @chmouel, the rationale for logging the error right here without returning was - the failure to add these results specific annotation is not critical to the functionality. So, we just log and continue creation of PR.

I hope [this change](https://github.com/openshift-pipelines/pipelines-as-code/pull/1245/commits/828b319f98979aba67fb81329d3eb273178904ec#diff-7d95c98719a2b5fe21a2104db3b95cda3425ab76566525da1437521470481c79R150) addresses your comment.